package.json

```diff
@@ -114,6 +114,7 @@
     "supertest": "4.0.2",
     "ts-node": "^9.1.1",
     "typescript": "^4.1.3",
+    "update-ts-references": "beta",
```

Comment:
I did publish this here https://registry.verdaccio.org/update-ts-references as `beta` while official is being published